src/Motor.Extensions.Hosting.RabbitMQ/RabbitMQMessageConsumer.cs

```diff
@@ -202,4 +161,96 @@ private void ConsumerCallback(BasicDeliverEventArgs args)
             _applicationLifetime.StopApplication();
         }
     }
+
+    private Dictionary<string, object> BuildQueueDeclareArguments(int? maxPriority, int? maxLength, long? maxLengthBytes, int? messageTtl, bool includeDeadLetterExchangeArguments = true)
```

Comment:
(nitpick) Wouldn't it make sense to provide `RabbitMQQueueLimitOptions` as the argument here instead of all of its members individually?